index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <main>
+    <div class="background-image"><img src="./images/Illustration.png"/ alt="Vector image"></div>
+    <header class="header-container">
+      <div class="header-title-container"><h2 class="header-title">John Doe</h2></div>
```

Comment:
Please consider making the logo section `John Doe` of your toolbar to be a link to the homepage, in most websites a logo that appears on the menu section always has a link to the homepage of the website. This also makes it easy for users to navigate your website with ease.